spack/packages/fiber/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+FFT_LIBS = (
+    'heffte',
+    'fftw',
+    'ffte',
+)
+
+
+class Fiber(CMakePackage):
+    """High Performance Fast Fourier Transform benchmark."""
+
+    homepage = "https://fiber.icl.utk.edu/"
+    git      = "https://github.com/icl-utk-edu/fiber"
+
+    maintainers = ['G-Ragghianti', 'luszczek']
+
+    version('master', branch='master')
+
+    variant('fft', default='none', values=FFT_LIBS,
+            multi=True, description='Supported FFT libraries')
+    #variant('onlycomplex', default=False)
```

Comment:
This will be flagged by Spack developers.

Reply:
What part?  We are still planning of heavy development before even thinking of submitting it to upstream spack.  This is why I configured this directory at a private spack repo (so it can easily be used from spack without haven't to be merged into upstream).